packages/@apphosting/adapter-angular/src/simple-server/server.ts

```diff
@@ -1,12 +1,24 @@
 import express from "express";
 import path from "path";
 import { fileURLToPath } from "url";
+import { readFileSync, existsSync } from "fs";
 
 const app = express();
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 const oneYear = 31_536_000_000;
 
+const prerenderedRoutesPath = path.join(__dirname, "..", "prerendered-routes.json");
+const prerenderedRoutes = existsSync(prerenderedRoutesPath)
+  ? JSON.parse(readFileSync(prerenderedRoutesPath, "utf8"))
+  : {};
+
+const staticWithoutCacheControl = express.static(path.join(__dirname, "..", "browser"));
```

Comment:
No cache control is a little agressive, maybe set to 5 minutes or something more reasonable